cmake/modules/dependency_notifier.cmake

```diff
@@ -0,0 +1,74 @@
+if(NOT CMAKE_C_COMPILER)
+    message("-- No C compiler was found. Please install the gcc package for your distribution:
+    DEB: apt-get install gcc
+    RPM: yum install gcc") 
+endif()
+
+if(NOT CMAKE_CXX_COMPILER)
+    message("-- No C++ compiler was found. Please install the g++ package for your distribution:
+    DEB: apt-get install g++
+    RPM: yum install gcc-c++") 
+endif()
+
+if(NOT libjson-c_FOUND)
+    message("-- json-c not found. Please install json-c package for you respective distribution:
+    DEB: apt-get install libjson0-dev
+    RPM: yum install json-c
+   If you have already installed this package in a nonstandard location 
+   please specify the location by defining the variable LIBJSON-C_ROOT in 
+   your cmake command as follows: cmake .. -DLIBJSON-C_ROOT=<path to install location>")
```

Comment:
This assumes the user is running cmake from a sub-directory of the git repo root. Maybe change `..` to `<path to clone dir>`?